surfstudio/StackOv
# Network services were upgraded

# Description
Added just simpler version of the network layer. From this moment services are responsible only for making requests and general configurations. For example, StackexchangeNetworkService gives us availability to make get requests to the stack exchange server with common query params.

# How has this been tested?
1. Open the app
2. Check that everything is fine

# Checklist


---

Closes #89
